uast/transformer/positioner/positions_test.go

```diff
@@ -65,23 +86,56 @@ func TestFillOffsetFromLineCol(t *testing.T) {
 	require.Equal(expected, out)
 }
 
-func TestEmptyFile(t *testing.T) {
+func TestFillOffsetEmptyFile(t *testing.T) {
 	require := require.New(t)
 
 	data := ""
 
 	input := nodes.Object{
-		uast.KeyStart: uast.Position{Line: 1, Col: 1}.ToObject(),
-		uast.KeyEnd:   uast.Position{Line: 1, Col: 1}.ToObject(),
+		uast.KeyStart: lineCol(1, 1),
+		uast.KeyEnd:   lineCol(1, 1),
 	}
 
 	expected := nodes.Object{
-		uast.KeyStart: uast.Position{Offset: 0, Line: 1, Col: 1}.ToObject(),
-		uast.KeyEnd:   uast.Position{Offset: 0, Line: 1, Col: 1}.ToObject(),
+		uast.KeyStart: fullPos(0, 1, 1),
+		uast.KeyEnd:   fullPos(0, 1, 1),
 	}
 
 	p := NewFillOffsetFromLineCol()
 	out, err := p.OnCode(data).Do(input)
 	require.NoError(err)
 	require.Equal(expected, out)
 }
+
+func TestPosIndex(t *testing.T) {
+	const source = `line1
```

Comment:
Please add a comment as to what's important about this example. Perhaps:

```
// Verify that a multi-byte Unicode rune does not displace offsets after
// its occurrence in the input.
```

or words to that effect.